cert/lib/cert/runner.rb

```diff
@@ -12,10 +12,14 @@ class Runner
     def launch
       run
 
-      installed = FastlaneCore::CertChecker.installed?(ENV["CER_FILE_PATH"], in_keychain: ENV["CER_KEYCHAIN_PATH"])
-      UI.message("Verifying the certificate is properly installed locally...")
-      UI.user_error!("Could not find the newly generated certificate installed", show_github_issues: true) unless installed
-      UI.success("Successfully installed certificate #{ENV['CER_CERTIFICATE_ID']}")
+      if Helper.mac?
+        UI.message("Verifying the certificate is properly installed locally...")
+        installed = FastlaneCore::CertChecker.installed?(ENV["CER_FILE_PATH"], in_keychain: ENV["CER_KEYCHAIN_PATH"])
+        UI.user_error!("Could not find the newly generated certificate installed", show_github_issues: true) unless installed
+        UI.success("Successfully installed certificate #{ENV['CER_CERTIFICATE_ID']}")
```

Comment:
[self-review] Im not sure why this message claims about installation. From my judgement, it only checks if its installed already in keychain or not.